### DISCUSSION TITLE
Someone finally wrote a real-time talking avatar that can be integrated into the textgen webui!

### DISCUSSION THREAD

**Discussion by sarutobiumon** [oobabooga/text-generation-webui#2378] [General]
Demo: https://www.youtube.com/watch?v=JUYIZD-p2w8
Code: https://github.com/jjmlovesgit/d-id_streams
@oobabooga kindly please consider integrating with the webui. Would be nice if someone can make it work with local LLM's.

**Comment by andreyarshavin229-cyber** [comment]
You can use free tool like tagshop
